docs/source/api.rst

```diff
@@ -606,3 +704,74 @@ Decimal methods
 
    DecimalValue.precision
    DecimalValue.scale
+
+.. _api.geospatial:
+
+Geo Spatial methods
```

Comment:
This is usually given as a single word "Geospatial"

Reply:
thanks @ian-r-rose I will fix that!